dashboard/app/entities.go

```diff
@@ -533,6 +540,21 @@ func (bug *Bug) getCommitInfo(i int) Commit {
 	return Commit{}
 }
 
+func (bug *Bug) increaseCrashStats(now time.Time) {
+	bug.NumCrashes++
+	date := timeDate(now)
+	if len(bug.DailyStats) == 0 || bug.DailyStats[len(bug.DailyStats)-1].Date < date {
+		bug.DailyStats = append(bug.DailyStats, BugDailyStats{date, 1})
+	} else if bug.DailyStats[len(bug.DailyStats)-1].Date == date {
+		bug.DailyStats[len(bug.DailyStats)-1].CrashCount++
+	}
+	// It is theoretically possible that this method might get into a situation, when
```

Comment:
It may be possible in small time windows due to some distributed inconsistencies maybe, but may also be very confusing that total and per-day stats don't match. E.g. e.g. we have 3 crashes in 2 days, 1 today and 1 yesterday. If any users will notice somebody will definitely rub our nose in it and ask to fix the bug we have.
I am thinking that it may be better to replace `else if` with just `else`. If the accounting to day is few seconds off, it looks better than mis-counting.